src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -73,7 +74,7 @@ public partial class DynamoView : Window, IDisposable
         // called on the view model and the process is not cancelled
         private bool isPSSCalledOnViewModelNoCancel = false;
         private readonly DispatcherTimer _workspaceResizeTimer = new DispatcherTimer { Interval = new TimeSpan(0, 0, 0, 0, 500), IsEnabled = false };
-
+        private ReadyParams sharedExtensionParams;
```

Comment:
adding a shared viewLoadedParams object for all viewExtensions so we can call dispose on it later.